src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -45,7 +45,13 @@
 class MysqlAdapter extends PdoAdapter implements AdapterInterface
 {
 
-    protected $signedColumnTypes = ['integer' => true, 'biginteger' => true, 'float' => true, 'decimal' => true, 'boolean' => true];
+    protected $signedColumnTypes = [
+        'integer' => true,
+        'biginteger' => true,
+        'float' => true,
+        'decimal' => true,
+        'boolean' => true
```

Comment:
```suggestion
        'boolean' => true,
```
Add comma for future diffs to be less noisy